.wp-env.json

```diff
@@ -2,6 +2,11 @@
 	"core": null,
 	"plugins": [ "." ],
 	"env": {
+		"development": {
+			"config": {
+				"WP_ENVIRONMENT_TYPE": "development"
+			}
+		},
```

Comment:
It looks like it's `local` out of the box, in wp env, so we might not even need this

```
% npm run env -- run cli wp eval 'echo wp_get_environment_type();'

> env
> wp-env run cli wp eval echo wp_get_environment_type();

ℹ Starting 'wp eval echo wp_get_environment_type();' on the cli container. 

local
```